### PR TITLE
Suspense

### PR DESCRIPTION
**Pull request by saifulislamdev**
So glad Gatsby released this version!

https://www.gatsbyjs.com/docs/reference/release-notes/v3.9/#react-18---new-suspense-ssr-architecture

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,8 +30,8 @@
     "netlify-cms-app": "^2.14.8",
     "node-sass": "^4.14.1",
     "prop-types": "^15.7.2",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
+    "react": "^18.0.0-alpha-419cc9c37-20210726",
```
Comment (aannyyaa) [inline package.json:33]: I'm not sure if we should be using an alpha version of react 18. There could be issues with using a version before general release? Can we use an older version of suspense with react 17?

**Review by aannyyaa — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (saifulislamdev) [reply package.json:33]: Yes, I understand what you're saying. However, older versions of `Suspense` aren't supported, not even the `Suspense` in version 17 of `React`.

Comment (aannyyaa) [reply package.json:33]: We could do a something like [this](https://www.gatsbyjs.com/docs/using-client-side-only-packages/#workaround-4-use-reactlazy-and-suspense-on-client-side-only)

Comment (saifulislamdev) [reply package.json:33]: In your example, we would still need `React.Suspense`, which won't be supported unless we upgrade.

**Pull request closed without merging**